modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -16,21 +26,120 @@
 			}
 
 			&-title {
+				display: flex;
+				flex-direction: row;
 				list-style: none;
 				font-size: var(--n-accordion-title-font-size);
 				padding: var(--n-accordion-title-padding);
 				border-width: var(--n-accordion-border-width);
 				border-color: var(--n-accordion-border-color);
 				border-style: var(--n-accordion-border-style);
+				justify-content: var(--n-accordion-item-title-justify-content);
+				flex-grow: var(--n-menu-item-title-flex-grow);
+
+				&-icon {
+					display: flex;
+					flex-direction: row;
+					align-items: center;
+					padding-inline: var( --n-accordion-title-icon-padding-inline );
+					order: var( --n-accordion-title-icon-order );
+
+
+					span > i {
+						color: var(--n-accordion-icon-color);
+					}
+
+					span > svg {
+						fill: var(--n-accordion-icon-color);
+					}
+				}
+
+				& > span {
+					cursor: pointer;
+				}
+			}
 
-				i {
-					color: var(--n-accordion-icon-color);
+			&[open] {
+				.e-n-accordion-item-title-icon-opened {
+					display: flex;
 				}
 
-				svg {
-					fill: var(--n-accordion-icon-color);
+				.e-n-accordion-item-title-icon-closed {
+					display: none;
+				}
+			}
+
+			&:not([open]) {
+
+				.e-n-accordion-item-title-icon-opened {
+					display: none;
+				}
+
+				.e-n-accordion-item-title-icon-closed {
+					display: flex;
 				}
 			}
 		}
 	}
 }
+
+//Responsive Mode
+@mixin title-styling-($device) {
```

Comment:
I would like to to Huddle about this. I would prefer to avoid the use of mixins, we will create bloated css code.

Reply:
Thanks for reaching out about this. Your suggestions has simplified the CSS.